transport/http/handler.go

```diff
@@ -82,6 +87,26 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	ctx := context.Background()
 
+	// Extract opentracing etc baggage from headers
+	// Annotate the inbound context with a trace span
+	tracer := h.Deps.Tracer()
```

Comment:
specifically, there's nothing in the Dependencies interface that suggests otherwise.